frontend/src/pages/MomentCreationPage.js

```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
         });
         this.setState({redirect: '/'});
     }
+
+    handleChange = (event) => {
+        console.log(event.target.files[0].name);
+        const filename = event.target.files[0].name;
+        document.getElementById('filename').innerHTML = filename;
+    }
+
     render(){
         return(
             <div>
                 <NavBar user={this.state.user}/>
+                <div className = "createMoment-header">
+                <img
+                    alt="UploadMoment"
+                    src={`http://${process.env.REACT_APP_IP}/res/uploadMoment.png`}
+                />
+                </div>
                 <div className = "CreateMoment">
                     {this.state.redirect && <Redirect to={this.state.redirect} />}
                     <form onSubmit={this.handleSubmit} encType="multipart/form-data" noValidate>
                         <div className = "content-container">
-                            <Header textSize={2} text="Title" />
-                            <input name="title" type="text" className="title" required/>
-                        </div>
-                        <div className = "content-container">
-                            <input type="file" name="file" required></input>
+                            <input type="file" name="file" id="file" className="inputfile" onChange={this.handleChange} required/>
+                            <label for="file">Choose Moment
+                                <img
+                                    alt="inputFile"
+                                    src={`http://${process.env.REACT_APP_IP}/res/inputfile.png`}
+                                />
+                            </label>
+                            <div id="filename"></div>
                         </div>
                         <div className = "content-container">
-                            <Header textSize={2} text="Description" />
-                            <textarea name="description" rows="8" cols="90"></textarea>
+                            <div className = "createMoment-description">
```

Comment:
This should be a `<label>`. Also, it should be placed right above the input field on the left side.
see https://uxdesign.cc/design-better-forms-96fadca0f49c